src/control/build/interop_test.go

```diff
@@ -160,6 +160,37 @@ func TestBuild_CheckCompatibility(t *testing.T) {
 				},
 			},
 		},
+		"custom matcher: narrowly-scoped rule (non-match)": {
+			a: testComponent(t, "server", "2.0.0"),
+			b: testComponent(t, "agent", "2.0.0"),
+			customRule: &build.InteropRule{
+				Self:        build.ComponentServer,
+				Other:       build.ComponentAgent,
+				Description: "server/agent 2.0.0/2.0.1 interop rule",
+				Match: func(self, other *build.VersionedComponent) bool {
+					v1_1_1 := build.MustNewVersion("2.0.0")
```

Comment:
Strikes me as a little weird for v2.0.0 to be named v1_1_1

Reply:
Changed the test, didn't update the variable names. Will fix.